rpc_jobs/multi_node_aio.yml

```diff
@@ -140,7 +140,7 @@
           }}
           instance_name = common.gen_instance_name()
           pubCloudSlave.getPubCloudSlave(instance_name: instance_name)
-
+          maas.add_maas_env_vars()
```

Comment:
Can you add a comment above this line to express what this is doing. It just makes the logic easier to follow later.